### ISSUE TITLE
Hyku v7.0.0 Featured Works images need alt tags

### ISSUE THREAD

**Issue by tlholt02** [samvera/hyku#2876]
Hyku v7.0.0 with Hyrax v5.2.0  Homepage (https://demo.hykudemo.org/)

The images under Featured Works need to have alt text. 

Example:
`<img suppress_link="true" src="https://demo.hykudemo.org/downloads/877aa5a7-8f20-492b-826c-295a87d0935b?file=thumbnail">` 
should be 
`<img alt="dog with cape" suppress_link="true" src="https://demo.hykudemo.org/downloads/877aa5a7-8f20-492b-826c-295a87d0935b?file=thumbnail">`

<img width="984" height="148" alt="screenshot" src="https://github.com/user-attachments/assets/aec9de16-02ca-4d66-b811-51412701438d" />
